aws/logs_monitoring/lambda_function.py

```diff
@@ -757,8 +757,14 @@ def parse_service_arn(source, key, bucket, context):
         # 2. We extract the loadbalancer name and replace the "." by "/" to match the ARN format
         # 3. We extract the id of the loadbalancer
         # 4. We build the arn
-        keysplit = key.split("_")
         idsplit = key.split("/")
+        # If AWSLogs isn't the first element of idsplit, there is a prefix 
+        # Remove the prefix before splitting they key
```

Comment:
The only edge case that I'd see is if the prefix is AWSLogs, in that case we might need an extra check on idsplit[1]
Otherwise LGTM.

Reply:
Updated!